src/components/generic/InputField/InputField.scss

```diff
@@ -1,3 +1,52 @@
+@mixin input-disabled {
```

Comment:
we should have the following states, which should each have their own set of css variables (no mixing of css variables from different states, because this will make it difficult to develop other themes):
- disabled
- static
- hover
- focus
- error-static
- error-hover
- error-focus

Reply:
the set of css variables should be (for each state):
--wxc-input-field--<state>--background
- border
- color
- box-shadow
- outline

Reply:
please use the same states for the mixin names as listed above (disabled, static, error-static etc.)

also, you can drop the 'input-' prefix if they are declared under `.#{$C} { ... }`, they will be local to this component.